podcastfy/config.yaml

```diff
@@ -3,7 +3,7 @@ output_directories:
   audio: "./data/audio"
 
 content_generator:
-  gemini_model: "gemini-1.5-pro-latest"
+  openai_model: "gpt-4o-mini"
```

Comment:
let's keep gemini as the default.

gpt-4o-mini has not been tested extensively
we use gemini 1.5 pro due to its massive context window of 2M token which allow for processing a list of input sources without trouble

we should consider generalizing to other llm backends here in a separate PR, we should use langchain for that purpose inside of implementing one backend at a time

Reply:
Thank you for pointing that out. I was testing on my local machine and didn’t switch it back. The reason I changed from Gemini to OpenAI was for extracting the "Thai" language, as OpenAI seemed to provide better translations in terms of context. However, it's not an issue—I’ll switch it back.